source/nanoFramework.Hardware.Esp32/TouchPad/TouchPadConfig.cs

```diff
@@ -0,0 +1,22 @@
+﻿//
+// Copyright (c) 2018 The nanoFramework project contributors
+// See LICENSE file in the project root for full license information.
+//
+
+namespace nanoFramework.Hardware.Esp32.TouchPad
+{
+	/// <summary>
+	/// Configuration parameters for TouchPad interrupt based class
+	/// </summary>
+	public class TouchPadConfig : TouchPadBaseConfig
+	{
+		//the threshold to trigger interrupt when the pad is touched
+		public float InterruptThresholdValue { get; set; }
+
+		public TouchPadConfig() : base()
+		{
+			// Use 2 / 3 of read value as the threshold to trigger interrupt when the pad is touched.
+			InterruptThresholdValue = 2 / 3;
```

Comment:
Care to double check this? SonarCloud is not happy about it because of the type.
Looking at the code, I'm not sure too if you meant to set it to 2/3 of the other value or really use a 2/3 as a constant...